pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -87,17 +87,30 @@ var (
 		  oc adm must-gather --image=my/image:tag --source-dir=/pod/directory -- myspecial-command.sh
 	`)
 
+	// According to https://github.com/openshift/enhancements/blob/master/enhancements/oc/must-gather.md#proposal
+	// we can assume /usr/bin/gather is always run in the end, even when a custom command is specified.
+	// That's why we always kill /usr/bin/gather, even though a custom command can be specified.
 	volumeUsageCheckerScript = `
-echo "volume percentage checker started....."
+echo "[disk usage checker] Started"
+target_dir="%s"
+usage_percentage_limit="%d"
 while true; do
-disk_usage=$(du -s "%s" | awk '{print $1}')
-disk_space=$(df -P "%s" | awk 'NR==2 {print $2}')
+disk_usage=$(du -sk "$target_dir" | awk '{print $1}')
```

Comment:
That would be better to revert the changes in here. Because these will be updated to fix https://issues.redhat.com/browse/OCPBUGS-56691.

Reply:
Do you mean to revert the whole line to the former state? Or something in particular?

Reply:
yes, reverting the changes related to `du` and `df` to former state.

Reply:
Amended.